### PR TITLE
Choose Web UI Links

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review: First MVC App](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-5.0&branch=pr-en-us-22968&tabs=visual-studio)
[Internal Review: Razor Pages Overview App](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/?view=aspnetcore-5.0&branch=pr-en-us-22968)
[Internal Review: Razor Pages Get Started](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-5.0&branch=pr-en-us-22968&tabs=visual-studio)

Creating an include that links to the Choose Web UI topic.  This will be included at the top level of getting started tutorials.

Related to #22635

### CODE REVIEW

**Comment by wadepickett** [comment]
@Rick-Anderson,  embarrassingly, I'm not understanding why I am getting a build fail here.  Do you see what I might be doing wrong?  The warnings are the usual alt text duplication warnings we get on multi-version docs like these.   The include shows up in the topics fine however as intended in the build.  Do I ignore the build fail or is there something I am missing that I can fix here?  Although it would be odd, it occurred to me that new include files might need to be published first before referencing them, even though they show up in the build, is that it?

Also, you wanted a link in the Blazor getting started, but the one we point to for getting started is on the Marketing/new customer site, not in docs, and not in this repo.  As an alternative, It doesn't feel right to mention this link in the to do list Blazor tutorial, which we do not promote as a starting point.  Thoughts?

**Comment by wadepickett** [comment]
@Rick-Anderson, please disregard my question about the build fail. We have repeating alt text warning for just about any of the multi-version topics but I don't remember the build failing over them, so I though I must have an issue with the way I did the include.  It is my first new include.  I guess it really is just the alt text issues.

Also, I tried to model (copy) after one of the existing includes, but I see those were not following the latest expected for metadata, so fixing that.

File: aspnetcore/tutorials/first-mvc-app/start-mvc.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 [!INCLUDE [consider RP](~/includes/razor.md)]
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/start-mvc.md:17]: Update this include
`
This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point. See [choose UI ...](_) which compares ... `

File: aspnetcore/tutorials/first-mvc-app/start-mvc.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
...
 
+[!INCLUDE [Choose web UI](~/includes/choose-ui-link.md)]
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/start-mvc.md:19]:
```suggestion
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by wadepickett** [comment]
close / reopen to force build.

**Comment by Rick-Anderson** [comment]
@wadepickett  once you get this error you can't eliminate it from close/open. It's a bogus error see #22992

File: aspnetcore/includes/razor.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ---
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 ---
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point.
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point. See <xref:tutorials/choose-web-ui>, which compares the benefits of ASP.NET Core UI functionalities.
```
Comment (Rick-Anderson) [inline aspnetcore/includes/razor.md:4]:
```suggestion
This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point. See <xref:tutorials/choose-web-ui>, which compares Razor Pages, MVC, and Blazor for UI development..
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@wadepickett  double check those warnings

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Some of those look like legitimate duplicates, not caused by monikers.

**Pull request merged**